src/components/PageElements/Elements/ReviewInlineInput.tsx

```diff
@@ -101,7 +101,7 @@ const ReviewInlineInput: React.FC<ReviewInlineInputProps> = ({
           <div className="review-button-container">
             <Button
               primary
-              disabled={isInvalidComment}
+              disabled={isInvalidComment || !reviewResponse.decision}
```

Comment:
Not technically part of this PR, but seemed like it should have been like this all along. Otherwise you can click the "Approve" button before selecting a decision, but nothing happens, which is confusing.